pkg/controller/cloneset/utils/cloneset_utils.go

```diff
@@ -142,6 +144,17 @@ func SplitPodsByRevision(pods []*v1.Pod, rev string) (matched, unmatched []*v1.P
 	return
 }
 
+func GroupUpdateAndNotUpdatePods(pods []*v1.Pod, updateRevision string) (update, notUpdate []*v1.Pod) {
```

Comment:
plz do not publish this function if it is not intended to be used by other package

Reply:
It does need to be published